### ISSUE TITLE
SPIKE: Playwright axe-core plugin integration

### ISSUE THREAD

**Issue by fzhao99** [CDCgov/dibbs-ecr-viewer#138]
In order to ensure our app is 508 compliant, we should build in accessibility checks into our Playwright e2e's. This ticket is to SPIKE out a plan for that. This Slack thread has more context: https://skylight-hq.slack.com/archives/C04T9HFFETX/p1726848001448139

There's a [playwright-axe](https://www.npmjs.com/package/@axe-core/playwright) library that we might want to use for these. If the package looks reasonable, a proof of concept would be welcome. Otherwise, look into other alternatives / options.
